### ISSUE TITLE
Misleading brick oven description and behavior

### ISSUE THREAD

**Issue by db47h** [CleverRaven/Cataclysm-DDA#66616]
### Describe the bug

TL;DR: the brick oven is built in three stages: brick oven base, exposed brick oven, and brick oven. I spent a long while using the exposed brick oven as a safe and smokeless fireplace and light source, but as soon as I managed to build the final stage, I could not use the oven as a fireplace anymore.

### Attach save file

Not relevant

### Steps to reproduce

Place an exposed brick oven: the player can drop wood into it and light a fire.
Build the final stage (door): if there is still a fire it will continue burning and provide light, however there is no way to feed the fire.

### Expected behavior

The description of the exposed says that it's not usable in this state, yet it can be used as a safe, smokeless fireplace. 

There are several options:

1. keep it working as is, but document the behavior in the description
2. make it work as a regular fire place even with the door. This might however not be very realistic. If I'm not mistaken these ovens worked by lighting a fire inside then use the accumulated heat to do the actual cooking (some references are available in #65660). 

Just keep in mind that as a player, I have no clue of which oven from the build menu will give me oven cooking 2. So with scarce resources, I will build a brick oven rather than a clay oven, without realizing that these are two very different things, seeing the brick oven just as a bigger version of the clay one, yet providing the same functionality with hopefully oven cooking 2.

### Versions and configuration

Cataclysm DDA experimental 37003bf75a58f6136077ecfbdc6e6d2be9bc28f2

**Comment by RenechCDDA** [comment]
I do remember intentionally not allowing the third stage to have items put in it but I am struggling to remember *why* I did that.

**Comment by RenechCDDA** [comment]
Hmm, no reasons coming to mind after giving it some thinking.

Well, easy to fix at least?

**Comment by db47h** [comment]
Yes, easy fix either way, but like I said, allowing it to be used as a fire place in the last stage might not be realistic.

**Comment by RenechCDDA** [comment]
In an ideal world I would like the completed oven to be two terrains - one unlit, one lit. There would be a minimum amount of combustibles needed, and it'd remain lit for only as long as it remained fed. 

The lit one would not actually be a fire, but merely a terrain that *provides the brick oven psuedo item*. The unlit one would not.

The problem with this approach(at the current moment) is that it would be leagues less efficient than existing ovens, where the baking costs are baked into the recipe.

**Comment by db47h** [comment]
I think the issue is that cooking recipes require a tool with the oven cooking quality AND some heat source (nearby fire or appliance with charges), making it hard to simulate this kind of oven in a way that is intuitive for the player, like the clay oven that provides both. An option would be to make it work like the bloomery (Innawood only?) that needs charges per use and make it completely unusable as a fire place.

**Comment by RenechCDDA** [comment]
The bloomery is furniture, the brick oven is terrain so it can be constructed in steps, and terrain sadly cannot work in such a way.

**Comment by db47h** [comment]
Plus it would not fulfill the heat source requirement. The best is probably to allow its use as a fireplace in the third stage.
